x-pack/legacy/plugins/epm/public/constants.ts

```diff
@@ -4,12 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { IconType } from '@elastic/eui';
-import { KibanaAssetType, AssetType, ServiceName } from '../common/types';
+import { ElasticsearchAssetType, KibanaAssetType, AssetType, ServiceName } from '../common/types';
 
 // TODO: figure out how to allow only corresponding asset types (KibanaAssetType, ElasticsearchAssetType)
 export const DisplayedAssets: Record<ServiceName, AssetType[]> = {
-  kibana: ['index-pattern', 'visualization', 'search', 'dashboard'],
-  elasticsearch: ['index-template', 'ingest-pipeline', 'ilm-policy'],
+  kibana: Object.values(KibanaAssetType),
```

Comment:
We use `Object.values(KibanaAssetType)` in 3 places in this PR. I wasn't sure if I wanted reuse a single exported variable (like `export const kibanaAssetTypes = Object.values(KibanaAssetType)` in, say, `common/constants`) or allow consumers to use the Object however they wish and deal with a little repetition.